client/src/App.js

```diff
@@ -1,14 +1,17 @@
 import React, { Component } from 'react'
-import './App.css';
+//import './App.css';
 import Login from './index';
 import Dashboard from './Dashboard';
+import Recipe from './Components/Recipes/Recipe.js';
 
 class App extends Component {
 
   render() {
     return (
       <div className="App">
-        <Dashboard/>
+        <Recipe Name="Barbeque Pecans" Image="https://icl.coop/wp-content/uploads/2015/10/BBQ-Pecans.jpg"/>
```

Comment:
nit: I believe the convention is that the attributes are lowercased. So `name` and `image` should be lowercased.